### DISCUSSION TITLE
Image can't invoke neither ImageFailed or ImageOpened events

### DISCUSSION THREAD

**Discussion by DJDQfff** [unoplatform/uno#13694] [Q&A]
I'm migrating a uwp app to uno , but meet some trouble .
There is a `GridView`with `Image`In `ItemTemplate`, but Image can't invoke `ImageFailed` or `ImageOpened `events on Android Head, Windows is OK.

As the Docs say , one of `ImageFailed `and `ImageOpened `shoulde be invoked.But actually none of them can be invoked.

I created a simple project to test , but it can work.

**Comment by limeniye** [comment]
Hello, @DJDQfff.
Thank you for the report.
Can yo send your simple project?

**Comment by DJDQfff** [comment]
sometime later, itis a complex project ,i need to simplify it



---Original---
From: ***@***.***&gt;
Date: Tue, Sep 19, 2023 18:49 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [unoplatform/uno] Image can't invoke neither ImageFailed orImageOpened events (Discussion #13694)




 
Hello, @DJDQfff.
 Thank you for the report.
 Can yo send your simple project?
